definitions/linkerd/component.yaml

```diff
@@ -0,0 +1,13 @@
+name: fabrikate-linkerd
+generator: static
+path: "./manifests"
+subcomponents:
+  - name: linkerd
+    generator: static
+    path: "./manifests"
+    hooks:
+      before-install:
+        - curl -sL https://run.linkerd.io/install | sh
```

Comment:
From the error in the CI/CD pipeline, it looks like we need to add the `--ignore-cluster` flag here.

```
time="2019-05-08T19:07:18Z" level=info msg="🎣  executing command: ~/.linkerd2/bin/linkerd install > manifests/linkerd.yaml"
time="2019-05-08T19:07:18Z" level=error msg="ERROR IN: /home/vsts/work/1/s/gopath/src/github.com/Microsoft/fabrikate-definitions/test"
time="2019-05-08T19:07:18Z" level=error msg="🚫  exit status 1\n"
time="2019-05-08T19:07:18Z" level=error msg="🚫  hook command failed with: Unable to build a Kubernetes client to check for configuration. If this expected, use the --ignore-cluster flag.\nError: invalid configuration: no configuration has been provided\n\n"
time="2019-05-08T19:07:18Z" level=info msg="💾  Loading /home/vsts/work/1/s/gopath/src/github.com/Microsoft/fabrikate-definitions/definitions/linkerd/component.yaml"
```